arch/src/x86_64/interrupts.rs

```diff
@@ -12,7 +12,7 @@ use std::sync::Arc;
 
 use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
 
-use kvm_bindings::kvm_lapic_state;
+use hypervisor::kvm_lapic_state;
```

Comment:
This should be `use hypervisor::x86_64::LapicState` and then replace all `kvm_lapic_state` instances with `LapicState`